fusion/example_video.py

```diff
@@ -0,0 +1,270 @@
+# Given a video sequence we want to run NNRT to extract motion
+# Usage python3 example_video.py <path to frames>
+
+import os
+import open3d as o3d
+import torch
+import numpy as np
+
+from utils import image_proc
+from model.model import DeformNet
+from model import dataset
+
+import utils.utils as utils
+import utils.viz_utils as viz_utils
+import utils.nnutils as nnutils
+import utils.line_mesh as line_mesh_utils
+import options as opt
+
+
+	#####################################################################################################
+	# Prepare data
+	#####################################################################################################
+
+	#####################################################################################################
+	# Source
+	#####################################################################################################
+	source_flat = np.moveaxis(source, 0, -1).reshape(-1, 6)
```

Comment:
Since `source` has never been defined in this file, this whole file is a dummy/template that isn't supposed to work just yet, correct?

Reply:
Yes, I will this file is actually not required. I shouldn't have uploaded it. I will delete it in the next PR. Thanks for letting me know. 

The main file is fusion.py

Reply:
I understand, thanks